tests/test_hash.c

```diff
@@ -176,9 +176,9 @@ static int do_sha512(void) {
 		return -2;
 	}
 	// hash with second state
-	if (msg_len > 64) {
+	if (msg_len > 128) {
```

Comment:
Would it be possible to use the `#define`d constants here? That way we'd be testing the definitions we have in the code.

Reply:
> Would it be possible to use the `#define`d constants here? That way we'd be testing the definitions we have in the code.

I'd been looking for suitable define's as the old code also didn't have any. Which ones do you have in mind?

Reply:
I don't think there are any constants exposed from the header files into this context. In which case I don't think it's worth the effort to add constants to expose just for this purpose.

Reply:
As per Douglas' comment above, left unchanged in https://github.com/open-quantum-safe/liboqs/pull/1454/commits/5f8d02c9c2651b9a5f3eca0364457a5b271a473a